src/style.css

```diff
@@ -0,0 +1,41 @@
+
+.page-body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
What about star in row is this need there ?

Reply:
I don't get what you want me to do(